.travis.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       python: "3.6"
       install: pip install pre-commit
       script: pre-commit run --all --show-diff-on-failure
-      after_success: true
-      before_install: true
+      after_success:
+      before_install:
```

Comment:
@lmignon FYI. This should clarify that after_success and before_install are overrides of the the global default, and not a boolean flag (it was actually running the true command).